### PR TITLE
Add the Awair Score as a gauge

### PR DESCRIPTION
**Pull request by anoldguy**
Very trivial patch to expose the Awair Score as a gauge.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -131,11 +132,19 @@ func (app *App) initializeGauges() {
...
+		Name:      "score",
+		Help:      "The current Awair Score",
+    })
+
 	app.TempGauge = tempGauge
 	app.HumidityGauge = humidityGauge
 	app.Co2Gauge = co2Gauge
 	app.VOCGauge = vocGauge
 	app.PM25Gauge = pm25Gauge
+    app.ScoreGauge = scoreGauge
```
Comment (jazware) [inline main.go]: Can you gofmt this file so it gets indented properly? Thanks!

**Comment by anoldguy** [comment]
Sorry about that!

**Review by jazware — APPROVED** [review]
LGTM

**Comment by jazware** [comment]
Thanks for the contribution! :)

**Pull request merged**